### PR TITLE
UX: updated account activation page design

### PR DESCRIPTION
**Pull request by MeghnaAJ**
Before:

Screenshot 2022-07-29 at 11 55 22

Screenshot 2022-07-29 at 11 55 34

After:

Screenshot 2022-07-28 at 22 19 33

Screenshot 2022-07-28 at 22 19 49

### CODE REVIEW

File: app/views/users/perform_account_activation.html.erb [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
-    <% else %>
-      <p><%= t('activation.please_continue') %></p>
-      <p><a class="btn" href="<%= path "/" %>"><%= t('activation.continue_button', site_name: SiteSetting.title) -%></a></p>
-      <%- content_for(:no_ember_head) do %>
-        <%= tag.meta id: 'data-auto-redirect', data: { path: path('/') } %>
-      <%- end %>
-      <%= preload_script 'auto-redirect' %>
-    <% end %>
+    <div class='activate-account'>
+      <h1 class="activate-title"><%= t 'activation.welcome_to', site_name: SiteSetting.title %> <img src="/images/wave.png" alt="" class="waving-hand"></h1>
```
Comment (CvX) [inline app/views/users/perform_account_activation.html.erb:8]: Too bad we can't our wavingHand logic here (and have to add the image file…)

Maybe we should convert it to an ember page. I don't think this needs to be rails-only? (cc: @davidtaylorhq)

Comment (davidtaylorhq) [reply app/views/users/perform_account_activation.html.erb:8]: > Maybe we should convert it to an ember page. I don't think this needs to be rails-only?

Yeah that would be great - we should do that. But I don't think it needs to block this redesign.

> our wavingHand logic here (and have to add the image file…)

Our wavinghand logic essentially provides a link to [/images/emoji/twitter/wave.png](https://meta.discourse.org/images/emoji/twitter/wave.png?v=12). We could do that on the server as well. @MeghnaAJ how large is the emoji being rendered? Would 72x72px be good enough?

Comment (davidtaylorhq) [reply app/views/users/perform_account_activation.html.erb:8]: According to the CSS it's 32px, so 72px should be more than enough.

So I think this should work

```suggestion
      <h1 class="activate-title"><%= t 'activation.welcome_to', site_name: SiteSetting.title %> <img src="<%= Emoji.url_for("wave") %>" alt="" class="waving-hand"></h1>
```

And then we don't need to add the extra PNG file.

Comment (MeghnaAJ) [reply app/views/users/perform_account_activation.html.erb]: I've made the suggested change. Thanks! 👍

**Review by CvX — APPROVED** [review]

**Pull request merged**